data/profiles/spaceweather/WSA.profile

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
The filename should be `wsa.profile` since we only use lowercase letters for filenames

Reply:
Well.. Other options are also Sun of Today, Todays Sun, Solar Magnetic Fields or something similar. Open to hear options

Reply:
Today's Sun sounds fine with me.  In which case the profile file name would be `todays_sun.profile` I guess